### ISSUE TITLE
Config Option: Persist Debug Log

### ISSUE THREAD

**Issue by maltfield** [BusKill/buskill-app#34]
Issue: Now that we're adding a shutdown trigger (#14), storing debug logs in temp storage doesn't always work out great (since they're gone after reboot).

This ticket will track the effort to add a config option to persist the debug log into the BusKill `DATA_DIR`

Of course, this is blocked by #16

**Comment by maltfield** [comment]
@Raihan93-coder would you be interested in adding this feature (after fixing bug #75)?

**Comment by Raihan93-coder** [comment]
Yes, I’d definitely be interested in working on that feature after resolving #75. Thanks for pointing me to it. :)

**Comment by maltfield** [comment]
This has been fixed by https://github.com/BusKill/buskill-app/pull/121
